utility/src/main/java/org/oppia/android/util/logging/ConsoleLogger.kt

```diff
@@ -33,6 +35,9 @@ class ConsoleLogger @Inject constructor(
    */
   val logErrorMessagesFlow: SharedFlow<ConsoleLoggerContext> = _logErrorMessagesFlow
 
+  /** Long-lived PrintWriter to improve performance by not reopening the file every time */
```

Comment:
We do not require documentation for private fields and functions, unless very necessary. In this case, it is safe to remove the KDoc.